questions/225/explanation.md

```diff
@@ -5,7 +5,7 @@ Then `main` is called, and we get to the line `X(object);`. This can be interpre
 2. As creating a new variable of type `X` named `object`. This is easier to see if you remove the parentheses, and it becomes just `X object`.
 
 §[stmt.ambig]¶1 in the C++ standard says:
-> An expression-statement with a function-style explicit type conversion (§8.2.3) as its leftmost subexpression can be indistinguishable from a declaration where the first declarator starts with a `(`. In those cases the statement is a declaration.
+> An expression-statement with a function-style explicit type conversion (§[expr.type.conv]) as its leftmost subexpression can be indistinguishable from a declaration where the first declarator starts with a `(`. In those cases the statement is a declaration.
```

Comment:
There's only an inline link for that reference. Should we omit such links or make them explicit like this?

Reply:
Yeah let's just omit those. We've already got more than enough work on our hands with the porting.